### ISSUE TITLE
Not selecting flying mount in Kalimdor/Azaroth/Northrend/Pandaria

### ISSUE THREAD

**Issue by Warloxx** [arkayenro/arkinventory#988]
Since the patch 7.3.5. the following applies:

> Expert Riding now unlocks the ability to fly in all zones prior to Draenor.
>
>The following requirements have been removed from the game:
>
>   Cold Weather Flying
>    Flight Master's License
>    Wisdom of the Four Winds
> Draenor Pathfinder and Broken Isles Pathfinder, Part Two still need to be completed to unlock flying.
[(Source)](https://www.mmo-champion.com/content/7329-Patch-7-3-5-Live-this-Week)

This makes this part:
```
				if zone == 1 or zone == 2 then
					
					if not GetSpellInfo( ( GetSpellInfo( 90267 ) ) ) then
						ArkInventory.Output( "azeroth but you dont know flight masters licence" )
						ArkInventory.IsFlyable = false
					end
					
				elseif zone == 4 then
					
					if not GetSpellInfo( ( GetSpellInfo( 54197 ) ) ) then
						
						ArkInventory.Output( "northrend but you dont know cold weather flying" )
						ArkInventory.IsFlyable = false
						
					end
					
				elseif zone == 6 then
					
					if not GetSpellInfo( ( GetSpellInfo( 115913 ) ) ) then
						
						ArkInventory.Output( "pandaria but you dont know wisdom of the four winds" )
						ArkInventory.IsFlyable = false
						
					end
```
from file _ArkInventoryLDB.lua_ (line 713-736) redundant and trigger a false negative for the old zones.

Also there is a missing qoute (") at the end of the line:
`ArkInventory.Output( "broken isles but you dont have the broken isles pathfinder part two achievement )`

**Comment by arkayenro** [comment]
fixed in 3.07.39

ps. missing quote doesnt matter, the lines are commented out

**Comment by Warloxx** [comment]
I uncommented those lines for debugging, that's why I noticed that some of those lines where missing the quotes.
